test/itemrenderer.cpp

```diff
@@ -546,3 +565,34 @@ TEST_CASE("item_renderer::get_feedtitle() returns item's feed URL "
 	const auto result = item_renderer::get_feedtitle(item);
 	REQUIRE(result == feedurl);
 }
+
+TEST_CASE("item_renderer::to_plain_text() escapes '<' for stfl",
```

Comment:
I don't think it should behave like this. `to_plain_text()` is used to save articles in .txt files, so it shouldn't contain STFL markup. The markup should only be present in the output of `to_stfl_list()`, which is used in `ItemViewFormAction`.

(This means the other test has to be changed as well.)

Reply:
Good catch. No idea why I didn't realize this myself when working on this.

The same issue already exists for the body of the articles (present before this PR, I will see if I can fix that at the same time).